bin/reth/src/dirs.rs

```diff
@@ -68,6 +75,19 @@ impl XdgPath for ConfigPath {
     }
 }
 
+/// Returns the path to the default reth import file.
+///
+/// Refer to [dirs_next::import_dir] for cross-platform behavior.
+#[derive(Default, Debug, Clone)]
+#[non_exhaustive]
+pub struct ImportPath;
+
+impl XdgPath for ImportPath {
+    fn resolve() -> Option<PathBuf> {
+        import_dir().map(|p| p.join("reth.toml"))
+    }
+}
```

Comment:
why not re-use `ConfigPath`?

Reply:
yeah `ConfigPath` can be re-used for this, will change